misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -324,11 +323,11 @@ class JettyService @Inject internal constructor(
     }
   }
 
-  override fun shutDown() {
-    val stopwatch = Stopwatch.createStarted()
-    logger.info("Stopping Jetty")
-
+  internal fun stop() {
     if (server.isRunning) {
+      val stopwatch = Stopwatch.createStarted()
```

Comment:
since this is called twice (harmlessly) when shutdown_sleep_ms is 0 we only want to log if we're still running